portfolio/src/main/webapp/index.html

```diff
@@ -43,6 +43,18 @@ <h2 class="greeting">Hi, I'm Karol Gutiérrez</h2>
 
       <!--COMMENTS-->
       <h2>Comments</h2>
+
+      <label for="comments-language">Choose a language:</label>
+      <select id="languages" onchange="getComments()">
+          <option value="">Original</option>
```

Comment:
In option, if you set SELECTED on one of these items, then it will be selected by default on page load.